src/display_device.cpp

```diff
@@ -819,9 +819,15 @@ namespace display_device {
 
     return DD_DATA.sm_instance->execute([](auto &settings_iface) {
       const auto devices {settings_iface.enumAvailableDevices()};
-      // If at least one device has additional info, it is active.
-      return std::any_of(std::begin(devices), std::end(devices), [](const auto &device) {
-        return static_cast<bool>(device.m_info);
+
+      // If there are no devices, then the API is probably not working...
+      if (devices.empty()) {
+        return false;
```

Comment:
Since we are checking if all devices are inactive... shouldn't this return true?

And the first early return should now return false?